<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configuracao</class>
 <widget class="QDialog" name="configuracao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Ajuste os atributos HSV até que a cor que deseja destacar seja a única parte isolada em branco.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="defaultButton">
      <property name="text">
       <string> Default Settings </string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>V_MAX</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>V_MIN</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>H_MAX</string>
      </property>
     </widget>
    </item>
    <item row="17" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Criar Jogador</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="hminBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="12" column="2">
     <widget class="QSpinBox" name="vminBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QSlider" name="horizontalSlider_6">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QSlider" name="horizontalSlider_4">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>S_MIN</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Name :</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QSpinBox" name="smaxBox">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="15" column="2">
     <widget class="QSpinBox" name="vmaxBox">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QSpinBox" name="sminBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>H_MIN</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QSlider" name="horizontalSlider_5">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QSlider" name="horizontalSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>S_MAX</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="hmaxBox">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Bola</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Azul</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Amarelo</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Time:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Escolha o time que deseja selecionar os jogadores.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Inverter</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Config</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hminBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hmaxBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sminBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smaxBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smaxBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sminBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hmaxBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hminBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vminBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vminBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vmaxBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vmaxBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
